shared/wallets/banner/index.tsx

```diff
@@ -48,7 +48,7 @@ const Banner = (props: Props) => (
         props.onAction && (
           <>
             {props.offerAdvancedSendForm === AdvancedBanner.receiverBanner &&
-              'This user accepts more assets than XLM. '}
+              'This person accepts other assets than XLM. '}
```

Comment:
"assets other than" reads better than "other assets than" to me. All the ways are weird. Could be a personal thing

Reply:
So "This person accepts assets other than XLM."? Will do

Reply:
Cool. Probably want to get an ok from design too as @cecileboucheron has worked on this wording already